frontend/isoko/src/components/businessPage/BusinessSidebar.tsx

```diff
@@ -0,0 +1,194 @@
+import React from 'react';
+import styled from 'styled-components';
+import StyledButton from '../../styles/StyledButton';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+   faClock,
+   faMapMarkerAlt,
+   faLink,
+} from '@fortawesome/free-solid-svg-icons';
+import { useNavigate } from 'react-router-dom';
+
+interface SidebarContainerProps {
+   width?: string;
+}
+
+const SidebarContainer = styled.div<SidebarContainerProps>`
+   width: ${(props) => (props.width ? props.width : '300px')};
+   background-color: #fefefe;
+   box-shadow: 0px 0px 1px rgba(0, 0, 0, 0.25), 0px 2px 4px rgba(0, 0, 0, 0.11);
+   border-radius: 5px;
```

Comment:
For all of these pixel values, should we be using em values instead so it is ratio-based rather than fixed?

Reply:
I just double checked with Cole and we're gonna be primarily using `em` values for font sizing, pixel sizing here is fine. Also I don't think we want the box shadow and border radius to scale with the screen